thirdparty_builtin/CMakeLists.txt

```diff
@@ -132,8 +132,13 @@ if(ENABLE_BENCHMARKS)
           message(FATAL_ERROR "Google Benchmark cannot be built when BUILD_SHARED_LIBS=ON or on Windows")
         endif()
 
-        add_subdirectory(gbenchmark-master-2017-05-19
-                         ${BLT_BUILD_DIR}/thirdparty_builtin/gbenchmark-master-2017-05-19)
+        if(CMAKE_CXX_STANDARD LESS 11)
```

Comment:
Should we not add `benchmark` if the `CMAKE_CXX_STANDARD` is less than `11`?  Just thought I'd throw that as an idea. I don't have a strong preference, since, I use C++11, but, I kind of see @davidbeckingsale point of view on this -- A build system should not impose the language standard. 

I would also add to that, that  a build system should not impose a language. Consider the case of a Fortran project using BLT for example. To do this properly, we should have proper language guards etc. when adding thirdparty libraries etc., but, that's probably another PR...

Reply:
The compiler may be defaulting to a high enough standard so that is why we went with a warning not an error. 

gbenchmark is OFF by default.  If you are turning it on you have to have C++.  It is a C++ library.